src/components/dashboard/FaceVerification/screens/IntroScreen.js

```diff
@@ -80,7 +80,7 @@ const IntroScreen = ({ styles, screenProps }) => {
   const [, checkForBrowserSupport] = useBrowserSupport({
     onSupported: requestCameraPermissions,
     onUnsupported: navigateToHome,
-    onlyIOS: true,
+    onCheck: () => (isIOSWeb ? isSafari : true),
```

Comment:
It's already included in the hook's logic
all you need regarding the task definition you exaplained me - is to pass unsupportedPopup: UnsuppportedDIalog in the SignupState to force show it both on IOS/other plarforms
```suggestion
```

Reply:
ok, I got what you mean